src/components/application_manager/include/application_manager/commands/command_request_impl.h

```diff
@@ -233,8 +260,8 @@ class CommandRequestImpl : public CommandImpl,
    * @return true if result code complies successful result code
    * otherwise returns false
    */
-  bool PrepareResultForMobileResponse(ResponseInfo& out_first,
-                                      ResponseInfo& out_second) const;
+  virtual bool PrepareResultForMobileResponse(ResponseInfo& out_first,
```

Comment:
@AByzhynar done in https://github.com/smartdevicelink/sdl_core/pull/1567/commits/9e2ce859407b0ff871832e4f6e99254cc2eb4a8f